packages/math-input/src/types.ts

```diff
@@ -39,27 +39,25 @@ export type IconConfig = {
 };
 
 export type NonManyKeyConfig = {
-    id: Keys;
-    type: KeyType;
+    id: Key;
+    type: Exclude<KeyType, "MANY">;
     icon: IconConfig;
     ariaLabel: string;
 };
 
-export type ManyKeyConfig = {
-    id: "MANY";
-    type: KeyType.MANY;
+export type ManyKeyConfig = Omit<NonManyKeyConfig, "type"> & {
+    type: Extract<KeyType, "MANY">;
```

Comment:
**question**: Why can't we just say: `type: "MANY"` here? 

I consulted https://www.typescriptlang.org/docs/handbook/utility-types.html but I'm none the wiser. 😖

Reply:
Well, we can. I just wanted to make the intent completely clear-- that this is a valid KeyType.